scpdt/plugin.py

```diff
@@ -19,12 +23,38 @@ def pytest_configure(config):
     doctest.DoctestModule = DTModule
     doctest.DoctestTextfile = DTTextfile
 
+def pytest_unconfigure(config):
```

Comment:
is this config a DTConfig instance or something from pytest?

Reply:
This is an instance of pytest's Config object